static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -60,6 +60,20 @@ type CalculateMessagesProps = {
   coupon?: Coupon
 }
 
+const messageForNotAttemptedExam = (course: Course) => {
+  let message =
+    "There are currently no exams available for scheduling. Please check back later."
+
+  if (course.can_schedule_exam) {
+    message = "Click above to schedule an exam with Pearson."
+  } else if (!R.isEmpty(course.exams_schedulable_in_future)) {
```

Comment:
I think @aliceriot would have asked to use `R.complement` here

Reply:
oh, either way is fine I think :)